Trim mountpoint input in Broadcast Preferences.

kevinwern opened the pull request:
Strip whitespace from beginning and end of string after applying/saving
changes.

daschuer commented:
That was fast, thank you. LGTM.

I like to push this to 1.12. Since GitHub does not allow to retarget a PR, please open a new one against 1.12.

kevinwern committed:
Trim mountpoint input in Broadcast preferences.

Strip whitespace from beginning and end of string after applying/saving
changes.
